src/main/java/eiteam/esteemedinnovation/boiler/ContainerBoiler.java

```diff
@@ -111,8 +112,8 @@ public void updateProgressBar(int id, int data) {
     }
 
     @Override
-    public boolean canInteractWith(@Nonnull EntityPlayer par1EntityPlayer) {
```

Comment:
Is this no longer `@Nonnull`?

Reply:
It isn't marked `@Nonnull` in the interface where it comes from. Also, we aren't using it.